docs/source/project_documentation/dev_guide/aac_validation.md

```diff
@@ -1,7 +1,11 @@
 # What is Validation in the AaC Language?
 Because the AaC DSL is leveraging plain-text YAML as the underpinning of the DSL, there is little to no functionality to guide users in the correctness of their YAML AaC structures. AaC has implemented a self-validating language feature so that users can reference which rules are applied to which AaC DSL components, and so that users can define validation for their own user-defined structures. To this end, AaC employs a plugin-based validator system where plugins provide Python-based validator implementations that can be referenced and applied to definitions in the AaC DSL.
 
-Validation rules in AaC are defined with the `validation` definition, which are required to have a corresponding implementation, called a validator plugin. This enables AaC's self-validating mechanism even though YAML is just a markup language.
+Data structure, or `schema`, validation rules in AaC are defined with the `validation` definition, which require a corresponding implementation, called a validator plugin. The validator plugin is what enables AaC's self-validating mechanism even though YAML is just a markup language. Each data structure is decomposed into its smaller structures, and then the structural constraints (validations) are applied. It is worth keeping in mind that, currently, we make no guarantees regarding the order in which validations are applied. 
```

Comment:
The part about "no guarantees regarding the order" begs the question -- in what case would this cause a problem? A simple example of a problem case would help a lot toward the reader's understanding of this concern. If we can not come up with such a problem case, is it really worth mentioning here?

Reply:
I am not certain of it really "causing a problem" perse. I took it to be more of a "heads up" that they might not execute in the order you expect them to: i.e. structure validation then primitive validation. Or even down to specific order within those types. And just because you create a new one, doesn't mean it would execute first or last because of being newest. 

I also made that assumption because that is my current understanding of validators.